hq/app/schedule/IamAudit.scala

```diff
@@ -36,30 +35,38 @@ object IamAudit extends Logging {
     }
   }
 
-  def makeCredentialsNotification(allCreds: Map[AwsAccount, Either[FailedAttempt, CredentialReportDisplay]]): List[Notification] = {
-    allCreds.toList.flatMap { case (awsAccount, eFCreds) =>
-      eFCreds match {
-        case Right(credsReport) =>
-          val outdatedKeys = outdatedKeysInfo(findOldAccessKeys(credsReport))
-          val missingMfa = missingMfaInfo(findMissingMfa(credsReport))
-          val targetGroups = getNotificationTargetGroups(outdatedKeys, missingMfa)
+  def getFlaggedCredentialsReports(allCreds: Map[AwsAccount, Either[FailedAttempt, CredentialReportDisplay]]): Map[AwsAccount, CredentialReportDisplay] = {
```

Comment:
I think in a future PR we could try to simplify this function further by getting to to take a `Map[AwsAccount, CredentialReportDisplay]` as parameter and dealing with the eithers somewhere else - might make it easier to test as well. out of scope for this work though!